prober/http_test.go

```diff
@@ -1686,7 +1688,39 @@ func TestSkipResolvePhase(t *testing.T) {
 
 		checkMetrics(expectedMetrics, mfs, t)
 	})
-	t.Run("With Proxy", func(t *testing.T) {
+	t.Run("With Proxy and resolve", func(t *testing.T) {
```

Comment:
is this test name correct? I see the `SkipResolvePhaseWithProxy: false` is false? same for the `With Proxy and without resolve` test case?

Reply:
sorry, that double negation is making it hard to read and reason about.

let's simplify the chained OR and AND condition where negation on already negated config `SkipResolvePhaseWithProxy`

Reply:
Test have been compacted.